velox/exec/GroupingSet.cpp

```diff
@@ -494,6 +494,9 @@ void GroupingSet::initializeGlobalAggregation() {
   lookup_->hits[0] = rows_.allocateFixed(offset, alignment);
   const auto singleGroup = std::vector<vector_size_t>{0};
   for (auto& aggregate : aggregates_) {
+    if (!aggregate.sortingKeys.empty()) {
```

Comment:
This is also part of the fix? thanks!

Reply:
Yes, we should also not call initializeNewGroups twice for sorted aggregations. This is not particularly harmful, but confusing and may cause issues in the future.